atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetricsTest.java

```diff
@@ -58,7 +67,12 @@ public void canUpdateConservativeMetrics() {
         assertTombstonesPutConservativeEquals(1);
         assertAbortedWritesDeletedConservativeEquals(2);
         assertSweepTimestampConservativeEquals(7);
-        assertLastSweptConservativeEquals(4);
+        assertLastSweptTimestampConservativeEquals(4);
+
+        puncherStore.put(3, 2);
+        puncherStore.put(4, 4);
```

Comment:
having wallClock = 4 for atlasTs = 4 is potentially confusing (ditto below with 6/6).